README.md

```diff
@@ -1 +1,79 @@
-# react-capstone-project
\ No newline at end of file
+# react-capstone-project
+In this project you can check for some nintendo characters, has a detail page for each character, this is a Single Page App Built with React, JavaScript and HTML,  it has been tested so you can also look for the testing files.
+
+## Built With:
+
+- Major languages: Javascript, HTML, CSS
+- Frameworks: React, Redux, Babel, CRA
+- Technologies used: VSC, Github
+
+## APIs used for this project : 
+
+- https://www.amiiboapi.com/
+
+## Live Demo (if available)
+
+- See live demo: https://maury-react-capstone.herokuapp.com/
+
+## Getting Started:
+- Clone the repository in your computer
+- npm install for modules
+- Run: npm start to see project displayed in your browser.
+
+### Prerequisites
+
+"Not available"
+
+### Setup
+
+"Not available"
+
+### Install
+
+- npm install (for node_modules)
+- npm install big.js
+
+### Usage
+
+"Not available"
+
+### Run tests
+
+run: npm test
+
+### Deployment
+
+
+
+## Authors
+
+👤 *Author1*
+
+- Mauricio Gallegos
+
+- GitHub: https://github.com/maury18
+- Twitter: https://twitter.com/MauryRodrguez6
+- LinkedIn:https://www.linkedin.com/in/mauricio-gallegos-rodr%C3%ADguez-380a96183/
+
+
+## 🤝 Contributing
+
+- I wanna thank to Nelson Sakwa for the letting me use his template as reference.
+- You can see the template here: https://www.behance.net/gallery/31579789/Ballhead-App-%28Free-PSDs%29
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] The link to your license is broken cause your project doesn't have any license. Please add a license to your project and I recommend using the [MIT license](https://choosealicense.com/licenses/mit/), Set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). another resource is [any other license] :+1: (https://choosealicense.com/licenses/) . :smile: